problem_builder/public/css/answer.css

```diff
@@ -1,6 +1,6 @@
 .mentoring .answer.editable {
     height: 250px;
-    width: 100%;
+    width: 99%;
```

Comment:
@itsjeyd Doesn't seem right. If there are some issues with it in edit mode, add a css rule to mentoring_edit or some css file only applied in studio.